front/package.json

```diff
@@ -3,15 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "type": "module",
+  "workspaces": [
+    "ui/base",
```

Comment:
What `ui/base` is used for ?

Reply:
See the section "Why keep front/ui/base/package.json?" in the PR description :P